modules/services/unison.nix

```diff
@@ -0,0 +1,127 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
```

Comment:
Typically the modules have a let-expression at top level setting a `cfg` variable. You can also put other useful variables there, like the `pairOf`. For example,

```nix
{ config, lib, pkgs, ... }:

with lib;

let

  cfg = config.services.unison;

  pairOf = t: with types; addCheck (listOf t) (l: length l == 2);

  pairOptions = submodule ({
    # …
  });

in

{
  meta.maintainers = with maintainers; [ pacien ];
  # …
}
```

Reply:
Moved up and amended.